eodag/utils/__init__.py

```diff
@@ -1360,3 +1361,39 @@ def parse_header(header):
     m = Message()
     m["content-type"] = header
     return m
+
+
+def create_point_bbox(
+    latitude: float, longitude: float, buffer: float = 0.00001
+) -> List[float]:
+    """
+    Creates a small bounding box around a given point specified by its latitude and longitude.
+
+    Parameters:
+    latitude (float): The latitude of the point.
+    longitude (float): The longitude of the point.
+    buffer (float): The size of the buffer to add/subtract to/from the latitude and longitude to create
+    the bounding box. Default is 0.00001.
+
+    Returns:
+    list: A list of four floats representing the bounding box in the Wekeo format
+    [min_lon, min_lat, max_lon, max_lat].
+    """
```

Comment:
please use sphinx docstring format https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html#the-sphinx-docstring-format